pitop/common/pt_os.py

```diff
@@ -1,10 +1,44 @@
+from dataclasses import dataclass, asdict
 from os import path
 from getpass import getuser
 
+
 ptissue_path = "/boot/pt-issue.txt"
 legacy_ptissue_path = "/etc/pt-issue"
 
 
+@dataclass
+class PitopOsBuildInfoV1:
+    schema_version: str
+    build_type: str
+    date: str
+    build_time_os_version: str
+    build_run_number: str
+    commit: str
+
+    @classmethod
+    def from_os_build_info(cls, build_info):
```

Comment:
I would say that default behaviour should be to try and parse the file automatically.